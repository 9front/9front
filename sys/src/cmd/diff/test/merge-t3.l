1
2
3
4
y
6
7
8
9
10
