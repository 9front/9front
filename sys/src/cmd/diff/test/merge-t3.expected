1
2
3
4
<<<<<<<<<< merge-t3.l
y
========== original
5
========== merge-t3.r
x
>>>>>>>>>>
6
7
8
9
10
